src/utils/index.js

```diff
@@ -134,6 +135,49 @@ function getData(emoji, skin, set, data) {
   return emojiData
 }
 
+function getEmojiDataFromNative(nativeString, set, data) {
+  if (data.compressed) {
+    uncompress(data);
+  }
+
+  const skinTones = ['', '🏻', '🏼', '🏽', '🏾', '🏿']
+  const skinCodes = ['', '1F3FB', '1F3FC', '1F3FD', '1F3FE', '1F3FF']
+
+  let skin
+  let skinCode
+  let baseNativeString = nativeString
+
+  skinTones.forEach((skinTone) => {
+    if (nativeString.indexOf(skinTone) > 0) {
+      const skinToneIndex = skinTones.indexOf(skinTone)
```

Comment:
We can figure out this `skinToneIndex` by using `skinTones.forEach((skinTone, i) => {}) instead